src/main/java/uk/gov/dvsa/enums/CertificateTypes.java

```diff
@@ -39,7 +38,9 @@ public enum CertificateTypes {
 
     TRAILER_INTO_SERVICE("CVS/TrailerIntoService"),
 
-    INSPECTION_CHECKLIST("MOT/InspectionChecklist");
+    INSPECTION_CHECKLIST("MOT/InspectionChecklist"),
+
+    IVA30_FAIL("CVS/IVA30_Fail");
```

Comment:
As discussed, I think this could just as easily be IVA30 or IVA_30 as IVA30 is specific to the fail (or more precisely the 'Notification of Refusal'), in the same way VTG30 and VTP30 are fails for annual tests for HGV/TRL and PSV respectively.